<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rastwidget</class>
 <widget class="QWidget" name="Rastwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>25</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="pushButton_close">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/grafiken.qrc">
       <normaloff>:/ewz/close_button.svg</normaloff>:/ewz/close_button.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MyComboBox" name="comboBox_Rast">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MySpinBox" name="spinBox_RastTemp">
     <property name="maximumSize">
      <size>
       <width>47</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="value">
      <number>88</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Â°C</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MySpinBox" name="spinBox_RastDauer">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>min</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_NachUnten">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../res/grafiken.qrc">
       <normaloff>:/global/dreieck_unten.svg</normaloff>:/global/dreieck_unten.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>7</width>
       <height>7</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_NachOben">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../res/grafiken.qrc">
       <normaloff>:/global/dreieck_oben.svg</normaloff>:/global/dreieck_oben.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>7</width>
       <height>7</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>./src/mycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MySpinBox</class>
   <extends>QSpinBox</extends>
   <header>./src/myspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/grafiken.qrc"/>
 </resources>
 <connections/>
</ui>
